staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2024,9 +2024,10 @@ type EnvVar struct {
 	// Escaped references will never be expanded, regardless of whether the variable
 	// exists or not.
 	// Defaults to "".
+	// You cannot set both value and valueFrom.
```

Comment:
Can we move this sentence to before the Defaults statement. Defaults statement is typically goes last.